gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -594,8 +594,9 @@ message EntitySelector {
   optional string route_id = 2;
   // corresponds to route_type in GTFS.
   optional int32 route_type = 3;
-  optional TripDescriptor trip = 4;
-  optional string stop_id = 5;
```

Comment:
For backwards compatibility `stop_id` needs to remain 5.  The new fields should be the first unused numbers in EntitySelector